### PR TITLE
chore: small improvement

### PR DESCRIPTION
**Pull request by soring323**

### CODE REVIEW

File: spec/README.md [hunk 1]
```diff
@@ -39,20 +39,20 @@ Our goal is to create a contract that enables users to trade their tokens smooth
 
 1. A taker takes an order by sending a `TakeSwap` transaction.
 2. The taker's sell tokens are sent to the escrow address owned by the contract.
-3. # if the taker's price is equate to order's price ,the contract sending Maker's selling tokens to taker and sending Taker's selling Token to maker. otherwise reject the take transaction and refund the taker's token.
+3. If the taker's price is equate to order's price ,the contract sending Maker's selling tokens to taker and sending Taker's selling Token to maker. otherwise reject the take transaction and refund the taker's token.
```
Comment (amityadav0) [inline spec/README.md:42]: we can remove this point, this is same as 6.

File: spec/README.md [hunk 2]
```diff
@@ -145,36 +145,27 @@ interface PlaceBidMsg {
...
-  completedAt: u64;
-  createdAt: u64;
+  expiredAt: number;
+  completedAt: number;
+  createdAt: number;
 }
 
 interface AcceptBidMsg {
   orderID: string;
 }
```
Comment (amityadav0) [inline spec/README.md:154]: we need bidder to identify bid, a single order can have multiple bids

File: spec/README.md [hunk 3]
```diff
@@ -145,36 +145,27 @@ interface PlaceBidMsg {
...
-interface TakeBidMsg {
-  orderID: string;
-  bidder: string;
-}
-```
-
 ```ts
 interface CancelBidMsg {
   orderID: string;
-  bidder: string;
```
Comment (amityadav0) [inline spec/README.md:168]: same, please restore it

File: spec/README.md (hunk 2 above)
Comment (soring323) [reply spec/README.md:154]: it repaced with msg.sender.

File: spec/README.md (hunk 3 above)
Comment (soring323) [reply spec/README.md:168]: same here

File: spec/README.md (hunk 2 above)
Comment (amityadav0) [reply spec/README.md:154]: that will be wrong, maker can accept order, but bids are stored based on bidder

Comment (soring323) [reply spec/README.md:154]: added

Comment (liangping) [reply spec/README.md:154]: > it repaced with msg.sender.

There is no limitation for a user to bid many times so far.

File: spec/README.md (hunk 3 above)
Comment (soring323) [reply spec/README.md:168]: don't need here. because only msg.sender can't cancel his bid.

File: spec/README.md [hunk 4]
```diff
@@ -39,20 +39,20 @@ Our goal is to create a contract that enables users to trade their tokens smooth
...
 
-1.  A maker cancels a previously created order. Expired orders can also be cancelled.
-2.  # An Order can only be cancelled once.
-3.  An order can only be cancelled once.
-4.  Tokens should be refunded when order is cancelled.
-5.  An order can be cancelled by the maker at any time, even if there are outstanding, incomplete bidding orders associated with it.
-6.  Maker should not be able to accept a bid order if the order has been canceled.
+1. A maker cancels a previously created order. Expired orders can also be cancelled.
+2. An Order can only be cancelled once.
+3. An order can only be cancelled once.
```
Comment (amityadav0) [inline spec/README.md]: reapeated, 2 and 3

File: spec/README.md (hunk 1 above)
Comment (amityadav0) [reply spec/README.md:42]: please remove this

**Pull request merged**